kratos/python/add_processes_to_python.cpp

```diff
@@ -235,8 +234,7 @@ void  AddProcessesToPython(pybind11::module& m)
     .def(py::init<ModelPart&, std::size_t>())
     ;
 
-//     py::class_<EntitiesEraseProcess<Node<3>>, EntitiesEraseProcess<Node<3>>::Pointer, Process>(m,"NodeEraseProcess") // TODO: Replace when the remainings of NodeEraseProcess have been cleaned up
-    py::class_<NodeEraseProcess, NodeEraseProcess::Pointer, Process>(m,"NodeEraseProcess")
+    py::class_<EntitiesEraseProcess<Node<3>>, EntitiesEraseProcess<Node<3>>::Pointer, Process>(m,"NodeEraseProcess")
     .def(py::init<ModelPart&>())
```

Comment:
please issue a deprecation warning telling that the `EntitiesEraseProcess` should be used instead (it is exposed to python I assume?)

Reply:
Well, it does the same, and the TODO has been there 4 years